README.md

```diff
@@ -83,4 +87,4 @@ For further information on contributing, reporting issues, suggesting features,
 
 ## Learn how to build with Apollo
 
-Check out the [Odyssey](https://odyssey.apollographql.com/) learning platform, the perfect place to start your GraphQL journey with videos and interactive code challenges. Join the [Apollo Community](https://community.apollographql.com/) to interact with and get technical help from the GraphQL community.
\ No newline at end of file
+Check out the [Odyssey](https://odyssey.apollographql.com/) learning platform, the perfect place to start your GraphQL journey with videos and interactive code challenges. Join the [Apollo Community](https://community.apollographql.com/) to interact with and get technical help from the GraphQL community.
```

Comment:
I'm not sure what the change is here?

Reply:
Yea I'm not sure why this is marked as a change because nothing is changed except line number.